lib/js/components/PopOver/PopOver.vue

```diff
@@ -208,6 +209,16 @@ export default {
 			type: String,
 			default: null,
 		},
+		appendToBody: {
+			type: Boolean,
+			default: false,
+		},
+		modifiers: {
+			type: Object,
+			default() {
+				return {};
```

Comment:
https://github.com/floating-ui/floating-ui/blob/v1.x/packages/popper/src/index.js#L40-L52
Empty object should be fine 👍